Autopilot/AttitudeManager/AttitudeManager.c

```diff
@@ -526,17 +529,17 @@ void imuCommunication(){
     imu_YawAngle = imuData[YAW];
     imu_PitchAngle = imuData[PITCH];
     imu_RollAngle = imuData[ROLL];
-#if DEBUG
+#if DEBUG && 0
```

Comment:
The code below it used to be commented out. I used it to debug the IMU, and this was easier.